mmv1/products/gkeonprem/VmwareAdminCluster.yaml

```diff
@@ -24,7 +24,7 @@ description: "A Google VMware Admin Cluster."
 exclude_delete: true
 id_format: "projects/{{project}}/locations/{{location}}/vmwareAdminClusters/{{name}}"
 import_format: ["projects/{{project}}/locations/{{location}}/vmwareAdminClusters/{{name}}"]
-timeouts: !ruby/object:Api::Timeouts
```

Comment:
😮

Reply:
Created https://github.com/GoogleCloudPlatform/magic-modules/pull/12848 to remove similar ruby/objects.

Reply:
and some other PRs to clean up other ruby references